app/views/financial_transactions/_form.html.erb

```diff
@@ -16,52 +16,55 @@
   <%= f.hidden_field :transactable_id %>
 
   <div class="form-group col-xs-12">
-    <%= f.label :amount, 'Amount', class: 'control-label col-xs-2' %>
+    <%= f.label :amount, "Amount", class: "control-label col-xs-2" %>
     <div class="col-xs-6">
-      <%= f.number_field :amount, class: 'form-control', min: 0, step: 0.01,
-        max: financial_transaction.rental.balance, value: financial_transaction.rental.balance, required: true %>
+      <%= f.number_field :amount, value: financial_transaction.rental.balance,
+                                  min: 0, max: financial_transaction.rental.balance,
+                                  class: "form-control", required: true %>
     </div>
   </div>
 
   <div class="form-group col-xs-12">
-    <%= label_tag :payment_type, 'Payment Form', class: 'control-label col-xs-2' %>
+    <%= label_tag :payment_type, "Payment Form", class: "control-label col-xs-2" %>
     <div class="col-xs-6">
-      <%= select_tag :payment_type, options_for_select(Payment.payment_types.keys), class: 'form-control', required: true %>
+      <%= select_tag :payment_type, options_for_select(Payment.payment_types.keys),
+                                    class: "form-control", required: true %>
     </div>
   </div>
 
   <div class="form-group col-xs-12">
-    <%= label_tag :contact_name, 'Contact Name', class: 'control-label col-xs-2' %>
+    <%= label_tag :contact_name, "Contact Name", class: "control-label col-xs-2" %>
     <div class="col-xs-6">
-      <%= text_field_tag :contact_name, nil, class: 'form-control', required: true %>
+      <%= text_field_tag :contact_name, nil, class: "form-control", required: true %>
     </div>
   </div>
 
   <div class="form-group col-xs-12">
-    <%= label_tag :contact_email, 'Contact Email', class: 'control-label col-xs-2' %>
+    <%= label_tag :contact_email, "Contact Email", class: "control-label col-xs-2" %>
     <div class="col-xs-6">
-      <%= text_field_tag :contact_email, nil, class: 'form-control', required: true %>
+      <%= text_field_tag :contact_email, nil, class: "form-control", required: true %>
     </div>
   </div>
 
   <div class="form-group col-xs-12">
-    <%= label_tag :contact_phone, 'Contact Phone', class: 'control-label col-xs-2' %>
+    <%= label_tag :contact_phone, "Contact Phone", class: "control-label col-xs-2" %>
     <div class="col-xs-6">
-      <%= text_field_tag :contact_phone, nil, class: 'form-control', placeholder: '5555555555', required: true %>
+      <%= text_field_tag :contact_phone, nil, placeholder: "5555555555",
```

Comment:
This placeholder is not consistent with the user form which displays a placeholder of 555-555-5555. I would like to see them perhaps both change to XXX-XXX-XXXX

Reply:
One could also use a mask. I'm sure there's a plugin available. But yeah, at least not a string of digits like this.